modules/rhcos-validate-live-media.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * installing/validation_and_troubleshooting/valiadting-an-installation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rhcos-validate-live-media_{context}"]
+
+= Validating {op-system} live media
+
+The {product-title} installation program contains pinned versions of {op-system} bootimages. Fully automated installations use these pinned artifacts by default. The mirror registry where you downloaded the installation program contains a `sha256sum` encrypted with the Red Hat product key.
+
+For user-provisioned infrastructure installations, you can access information and use the {product-title} installer to indirectly validate {op-system} bootimage artifacts using their SHA-256 checksums.
+
+.Procedure
+
+* Run the following command to print the metadata for any bootimage artifact:
++
+[source,terminal]
+----
+$ openshift-install coreos print-stream-json | jq <bootimage> <1>
+----
+<1> The query for the bootimage you want to obtain information on. For validation purposes, the bootimage artifact must have a generated `sha256sum`. This can include OVA, VHD, QCOW2 and others. For example, to get an `x86_64` architecture `iso` file for bare metal platforms, this value would be `.architectures.x86_64.artifacts.metal.formats.iso`.
```

Comment:
```suggestion
<1> The query for the bootimage you want to obtain information. For validation purpose, the bootimage artifact must have a generated `sha256sum`. This can include OVA, VHD, QCOW2 and others. For example, to get an `x86_64` architecture `iso` info for bare metal platforms, this value would be `.architectures.x86_64.artifacts.metal.formats.iso`.
```